dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -126,6 +126,24 @@ def _check_iteration_stop(self):
                 self.reset()
             raise StopIteration
 
+
+def non_empty(shape):
+    for s in shape:
+        if s == 0:
+            return False
+
+    return True
+
+def get_mx_array(shape, ctx=None, dtype=None):
+    # mx.nd.empty doesn't support np.longlong, but it does np.int64
```

Comment:
I'd say that it is a bug in mxnet, as it maps np.longlong to np.int64 when using zeros.
Maybe a TODO to remove it at some point would be nice.